examples/cpp/content_filter/HelloWorld.hpp

```diff
@@ -80,9 +80,9 @@ class HelloWorld
     eProsima_user_DllExport HelloWorld(
```

Comment:
Please, ensure that the types and their type objects are generated with the latest `fastddsgen` release

Reply:
Yes the types are generated with the latest version of `fastddsgen`